src/app/components/Results/ResultsList.jsx

```diff
@@ -136,12 +137,20 @@ class ResultsList extends React.Component {
     const totalItems = items.length;
     const hasRequestTable = items.length === 1;
 
+    let bibUrl = `${appConfig.baseUrl}/bib/${bibId}`
+
+    if (this.props.searchKeywords) {
+      bibUrl += `?searchKeywords=${this.props.searchKeywords}`;
+    } else if (Store.getState().searchKeywords) {
+      bibUrl += `?searchKeywords=${Store.state.searchKeywords}`;
+    }
+
```

Comment:
Here's an idea, instead of passing in the string `"undefined"` (in case the prop or state are empty), just don't pass anything in. In the breadcrumbs function, check if there's a value at all instead of checking for that string.

```
const searchKeywords = this.props.searchKeywords || Store.getState().searchKeywords);
let bibUrl = `${appConfig.baseUrl}/bib/${bibId}`;

if (searchKeywords) {
  bibUrl += `?searchKeywords=${searchKeywords}`;
} 
```